publish-common/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
with mockito 5 this configuration is not needed

Reply:
Uplifted to Mockito 5.3.1; removed mockito-inline.